src/Stratis.Bitcoin.Features.PoA/PoAMiner.cs

```diff
@@ -164,9 +156,6 @@ private async Task CreateBlocksAsync()
                     builder.AppendLine($"Block was mined {chainedHeader}.");
                     builder.AppendLine("<<==============================================================>>");
                     this.logger.LogInformation(builder.ToString());
-
-                    int halfTargetSpacingMs = (int)this.network.ConsensusOptions.TargetSpacingSeconds * 1000 / 2;
-                    await this.WaitBeforeCanMineAsync(halfTargetSpacingMs, this.cancellation.Token).ConfigureAwait(false);
```

Comment:
Also removed this, superfluous, don't need it when it would always loop over and enter the wait elsewhere anyway